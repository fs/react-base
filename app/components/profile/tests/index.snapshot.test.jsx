import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { fakeSession } from 'mocks/fakeSession';
import Profile from '../';

describe('Profile', () => {
  let props;
  const renderComponent = () => shallow(<Profile {...props} />);

  beforeEach(() => {
    props = fakeSession;
  });

  it('renders correctly', () => {
    expect(toJson(renderComponent())).toMatchSnapshot();
  });
});
