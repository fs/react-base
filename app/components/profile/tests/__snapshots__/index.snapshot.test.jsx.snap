// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile renders correctly 1`] = `
<Grid
  bsClass="container"
  componentClass="div"
  fluid={false}
>
  <h1>
    test@test.com
  </h1>
  <p>
    This page created to demonstrate ability to protect route. This page available only for authenticated users.
  </p>
</Grid>
`;
